inc/options/objectcache.php

```diff
@@ -72,13 +72,13 @@
 
 			<tr>
 				<th colspan="2">
-					<?php $this->checkbox( 'objectcache.enabled_for_wp_admin' ) ?> Enable caching for wp-admin requests</label>
+					<?php $this->checkbox( 'objectcache.enabled_for_wp_admin' ) ?>( ' Enable caching for wp-admin requests', 'w3-total-cache' )</label>
 					<p class="description"><?php _e( 'Enabling this option will increase wp-admin performance, but may cause side-effects', 'w3-total-cache' )?></p>
 				</th>
 			</tr>
 			<tr>
 				<th colspan="2">
-					<?php $this->checkbox( 'objectcache.fallback_transients' ) ?> Store transients in database</label>
+					<?php $this->checkbox( 'objectcache.fallback_transients' ) ?>( ' Store transients in database', 'w3-total-cache' )</label>
```

Comment:
Not a translation function.  Add PHP tags and a semi-colon too.  Might as well concatenate using ` . `.